nebula/ui/components/forms/VPNTextField.qml

```diff
@@ -36,6 +36,26 @@ TextField {
     leftPadding: VPNTheme.theme.windowMargin
     rightPadding: VPNTheme.theme.windowMargin
 
+    cursorDelegate: Rectangle {
+        // force cursor color and blink
+        id: cursor
+        visible: parent.activeFocus && !parent.readOnly && parent.selectionStart === parent.selectionEnd
+        color: textField.placeholderTextColor
+        width: 1
+
+        Timer {
+            // This is duped from CursorDelegate.qml to preserve cursor blinking which is overwritten
```

Comment:
Why do we need this? Is it a bug in qt?

Reply:
Yes, the cursor blink disappears entirely after customizing cursorDelegate.

Reply:
do you know if there is already a qt-bug? If no, let's file one. Then we can write the URL of the bug here.

Reply:
We can land this and we can write the bug ID as a follow-up. What do you think?